src/main/java/com/revealprecision/revealserver/api/v1/facade/factory/TaskFacadeFactory.java

```diff
@@ -62,5 +63,31 @@ private static String getStructureId(Task task, Action action ){
     return structureId;
   }
 
-
-}
+  public static TaskFacade getTaskFacadeObj(String requester, String taskPlanParentId, TaskEvent task) {
+    return TaskFacade.builder()
```

Comment:
what's the difference between getTaskFacadeObj and getEntity they both seem to be building the same TaskFacade object. wont we end up with different results depending on which one is called?

If one is used by kafka, at least we need to extact the common fields being set